test-browser/index.cfm

```diff
@@ -30,8 +30,9 @@
 	qResults = directoryList( targetPath, false, "query", "", "name" );
 	// Get the back path
 	if( len( url.path ) ){
-		backPath = replacenocase( url.path, listLast( url.path, "/" ), "" );
-		backPath = reReplace( backpath, "/$", "" );
+		backPath = ListToArray( url.path, "\" );
```

Comment:
This is great, but this won't account for *nix based systems.  YOu wil have to use the second delimiter to make this compliant with both windows and *nix based systems.

```js
backPath = ListToArray( url.path, "\/" );
```

Please add that to all ocassions!